policy/modules/system/systemd.te

```diff
@@ -1158,6 +1165,34 @@ seutil_read_file_contexts(systemd_sessions_t)
 
 systemd_log_parse_environment(systemd_sessions_t)
 
+
+#########################################
+#
+# Sysusers local policy
+#
+
+allow systemd_sysusers_t self:capability { chown fsetid };
+allow systemd_sysusers_t self:process setfscreate;
+allow systemd_sysusers_t self:unix_dgram_socket sendto;
+
+dev_write_kmsg(systemd_sysusers_t)
```

Comment:
This should be handled by `systemd_log_parse_environment()`

Reply:
I may have messed up git, I'll fix this.